gridstatus/ercot_api/ercot_api.py

```diff
@@ -978,6 +990,81 @@ def get_spp_day_ahead_hourly(self, date, end=None, verbose=False):
 
         return data.sort_values(["Interval Start", "Location"]).reset_index(drop=True)
 
+    @support_date_range(frequency=None)
+    def get_dam_load_and_gen_60_day_resources_as_offers(
```

Comment:
I think it makes sense to have this one method return two dataframes in a dictionary because:

* Both files are in the same zip file so it's more efficient to get both files at once
* The datasets are closely related
* This is the same approach as in `Ercot().get_60_day_dam_disclosure`